src/manager/msThemeManager.ts

```diff
@@ -197,11 +197,8 @@ export class MsThemeManager extends MsManager {
         return this.themeSettings.get_boolean('clock-app-launcher');
     }
 
-    getPanelSize(monitorIndex: number) {
-        return (
-            this.themeSettings.get_int('panel-size') *
-            global.display.get_monitor_scale(monitorIndex)
-        );
+    getPanelSize() {
+        return this.getScaledSize(this.themeSettings.get_int('panel-size'));
```

Comment:
Why does this no longer depend on the monitor index? You can have different dpi scaling on different monitors, can you not?

Reply:
On X11 you cannot and on Wayland it does work properly only if you have fractionnal scaling enabled by ` gsettings set org.gnome.mutter experimental-features "['scale-monitor-framebuffer']"` and with this setting enabled the UI scaling factor is set to `1` and the scaling is automatically managed by Clutter

Reply:
Great! Add a comment :)